src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -1529,6 +1565,30 @@ public static IEdmTypeDefinitionReference GetUInt64(this EdmModel model, string
                 isNullable);
         }
 
+        /// <summary>
+        /// Get the <see cref="IEdmTargetPath"/> from a target path string.
+        /// </summary>
+        /// <param name="model">The EdmModel.</param>
+        /// <param name="targetPath">Target path containing segments separated by '/'. For example: "A.B/C/D.E/Func1(NS.T,NS.T2)/P1".</param>
+        /// <param name="caseInsensitive">Property name case-insensitive or not.</param>
+        /// <returns>The created <see cref="IEdmTargetPath"/>.</returns>
+        public static IEdmTargetPath GetTargetPath(this IEdmModel model, string targetPath, bool caseInsensitive = true)
+        {
+            EdmUtil.CheckArgumentNull(model, nameof(model));
+            EdmUtil.CheckArgumentNull(targetPath, nameof(targetPath));
+
+            string[] targetPathSegments = targetPath.Split('/');

```

Comment:
Is the `targetPath` string here the exact path that would be generated by `GetPathString()`?

Reply:
Yes